doc/develop/west/zephyr-cmds.rst

```diff
@@ -174,3 +174,20 @@ As does deleting them::
 Additionally the tool allows you to specify the modules you want to list,
 fetch or clean blobs for by typing the module names as a command-line
 parameter.
+
+.. _west-twister:
+
+Twister wrapper: ``west twister``
+*****************************************
```

Comment:
Align `*` with the length of the section by removing extra characters.

Reply:
done